test/benchmarks/OptimizorClub.sol

```diff
@@ -0,0 +1,782 @@
+            '<animate additive="sum" attributeName="startOffset" from="0%" to="100%" begin="0s" dur="30s"'
+            ' repeatCount="indefinite"/></textPath><textPath startOffset="-50%" fill="#fff" font-family="\'Courier New\', monospace" font-size="10px" xlink:href="#text-path-a">',
+            projectName,
+            unicode" • ",
+            solverAddr,
+            '<animate additive="sum" attributeName="startOffset" from="0%" to="100%" begin="0s" dur="30s" repeatCount="indefinite"/></textPath></text>'
+        );
+    }
+    function generateSVGCardMantle(string memory challengeName, uint32 rank, uint32 participants)
+        private
+        pure
+        returns (string memory svg)
+    {
+        svg = string.concat(
+            '<g mask="url(#fade-symbol)"><rect fill="none" x="0" y="0" width="290" height="200"/><text y="70" x="32" fill="#fff" font-family="\'Courier New\', monospace" font-weight="200" font-size="28px">',
+            challengeName,
+            '</text><text y="115" x="32" fill="#fff" font-family="\'Courier New\', monospace" font-weight="200" font-size="20px">'
+            "Rank ",
+            LibString.toString(rank),
+            " of ",
+            LibString.toString(participants),
+            "</text></g>"
+        );
+    }
+    function generateRankBorder(uint32 rank) private pure returns (string memory svg) {
+        string memory color;
+        if (rank == 1) {
+            color = "rgba(255,215,0,1.0)";
+        } else if (rank == 2) {
+            color = "rgba(255,255,255,1.0)";
+        } else if (rank == 3) {
+            color = "rgba(205,127,50,1.0)";
+        } else {
+            color = "rgba(255,255,255,0.2)";
+        }
+        svg = string.concat(
+            '<rect x="16" y="16" width="258" height="468" rx="26" ry="26" fill="rgba(0,0,0,0)" stroke="', color, '"/>'
+        );
+    }
+    function generateSvgCurve(string memory challengeSVG) private pure returns (string memory svg) {
+        svg = string.concat('<g mask="url(#none)"', ' style="transform:translate(30px,130px)">', challengeSVG, "</g>");
+    }
+    function generateSVGPositionDataAndLocationCurve(string memory tokenId, uint256 gasUsed, uint256 gasOpti)
+        private
+        pure
+        returns (string memory svg)
+    {
+        string memory gasUsedStr = LibString.toString(gasUsed);
+        string memory gasOptiStr = LibString.toString(gasOpti);
+        uint256 str1length = bytes(tokenId).length + 4;
+        uint256 str2length = bytes(gasUsedStr).length + 10;
+        uint256 str3length = bytes(gasOptiStr).length + 14;
+        svg = string.concat(
+            '<g font-family="\'Courier New\', monospace" font-size="12" fill="#fff">'
+            '<g style="transform:translate(29px, 384px)">' '<rect width="',
+            LibString.toString(uint256(7 * (str1length + 4))),
+            '" height="26" rx="8" ry="8" fill="rgba(0,0,0,0.6)"/>' '<text x="12" y="17"><tspan fill="#999">ID: </tspan>',
```

Comment:
@leonardoalt Check this out.
```suggestion
            '" height="26" rx="8" ry="8"/>' '<text x="12" y="17"><tspan fill="#999">ID: </tspan>',
```
Magic. It compiles via legacy now :P

With that maybe we can go back to the simpler version of `run.sh` and keep assuming that benchmarks have to compile?

Reply:
Interestingly, this works too, and lets us avoid having to modify the content of the string:
```suggestion
            string.concat('" height="26" rx="8" ry="8" fill="rgba(0,0,0,0.6)"/>' '<text x="12" y="17"><tspan fill="#999">ID: </tspan>'),
```

Reply:
Well, the first one has a different value, but yea the second is weird haha

Reply:
Yeah, that's even better, so that's what I'll do. Ultimately, the idea behind this script is to compare legacy vs via-ir compile times, so having both pass compilation successfully is ideal :)

Reply:
> Ultimately, the idea behind this script is to compare legacy vs via-ir compile times, so having both pass compilation successfully is ideal :)

Yeah. That's why for our use case even modifying the string would have been acceptable. But nice that we don't have to.

BTW, it would look less weird if we replaced this new `string.concat()` call with a local variable.

Reply:
Not worth the effort honestly, it's not our code anyway.